kinto/__main__.py

```diff
@@ -77,9 +85,9 @@ def main(args=None):
 
         # Install postgresql libraries if necessary
         if backend == "postgresql":
-            try:
+            try:  # pragma: no cover
```

Comment:
This is [supposed to be tested](https://github.com/Kinto/kinto/pull/576/files#diff-6e50bc18504011fcabd48bb239812b9fR60) but I don't know why the coverage is not calculated.